hnsecure/src/main/java/ca/bc/gov/hlth/hnsecure/Route.java

```diff
@@ -55,6 +61,8 @@ public void configure() {
             
             //dispatch the message based on the receiving application code and message type
             .choice()
+            	.when(simple("${body} contains 'AR'"))
+            	.log("There is an error validation the message and it will not be sent to endpoint")
```

Comment:
We should implement the code to handle these validation errors as part of this story. 

I think the best way to do this is to throw a custom exception from the Validator and to have an `onException().handled(true)` handler in the Route.java that catches the validation exception.